dashboard/settings.py

```diff
@@ -181,14 +181,24 @@
 LOGGING = {
     'version': 1,
     'disable_existing_loggers': False,
+    # Gunicorns logging format https://github.com/benoitc/gunicorn/blob/19.x/gunicorn/glogging.py
+    'formatters': {
+        "generic": {
+            "format": "%(asctime)s [%(process)d] [%(levelname)s] %(message)s",
+            "datefmt": "[%Y-%m-%d %H:%M:%S %z]",
```

Comment:
I feel we should be log time in the EST than UTC

Reply:
I think it was logging in the timezone of the server, which is set to Eastern time in the Dockerfile, I can double check this though. The %z just confirms what time zone it's in.